### DISCUSSION TITLE
Stuck on run_uproot_job

### DISCUSSION THREAD

**Discussion by yanfr0818** [CoffeaTeam/coffea-casa#132] [UNL tech support] [answered]
Hi there,

I'm having trouble with opening files with uproot. For example, in example1.ipynb:

from dask.distributed import Client
import time

client = Client("tls://localhost:8786")

fileset = {'SingleMu' : ["root://eospublic.cern.ch//eos/root-eos/benchmark/Run2012B_SingleMu.root"]}

output = processor.run_uproot_job(fileset,
                                treename = 'Events',
                                processor_instance = Processor(),
                                executor = processor.dask_executor,
                                executor_args = {'schema': processor.NanoAODSchema, 'client': client}
                                )

> [                                        ] | 0% Completed | 16min 58.4s

**Comment by oshadura** [comment, marked as answer]
hi @yanfr0818, sorry for this I was just changing image and you got unlucky and using not working one. Could I ask you please to restart a server? For this try in menu 'File'->"Hub Control Panel"-> "Stop Server" -> "Start Server". Thanks!

**Reply by yanfr0818** [reply]
Thanks for letting me know. It's working now.

**Comment by oshadura** [comment]
Also just in case for the future, on the left corner where is Dask Labextention server (see screenshot) you could notice that there is something wrong before, when you see that the number of workers available for user is 0 (it should be least 1 worker by default)
<img width="337" alt="Screenshot 2021-06-14 at 19 19 01" src="https://user-images.githubusercontent.com/7012420/121932841-a0d9d500-cd45-11eb-8253-becc87e36e9d.png">

**Reply by yanfr0818** [reply]
I noticed that. But the odd thing was that the kernel could handle basic things like importing python package, which leading me to think that was about uproot...
